blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,13 @@ Unreleased Changes
 Changes
 =======
 
+- Changed how columns of type :ref:`geo_point_data_type` are being communicated
+  to PostgreSQL clients: Before clients were told that those columns are double
+  arrays, now they're correctly mapped to the PostgreSQL ``point`` type. This
+  means that applications using clients like ``JDBC`` will have to be adapted
+  to use ``PgPoint``. (See `Geometric DataTypes in JDBC
```

Comment:
Isn't it a breaking change in this case? @mfussenegger

Reply:
Ah just saw the summary. So in this case it's not a breaking change..

Reply:
I wasn't sure if it should go into the breaking changes section. It is a bit of a grey area in this case as it could also be regarded as a fix (in that the previous mapping didn't make so much sense).

Maybe it would make sense to move it to give it a bit more visibility since it is affecting client applications.

Reply:
Ok, then let's better be safe than sorry.